File: resources/array.cpp
#include <iostream>
using namespace std;

void printArray(int array[], int length)
{
  for (int i = 0; i < length; i++) {
    cout << array[i] << " ";
  }
  cout << endl;
}

int main() {
  int foo[] = {1, 3, 3, 7};
  printArray(foo, 4);
}

File: resources/array.js
function printArray(array)
{
    var length = array.length;
    var i = 0, str = "";

    for (i = 0; i < length; i++) {
        str += array[i] + " ";
    }
    console.log(str);
}


var foo = [1, 3, 3, 7];
printArray(foo, 5);


